README.md

```diff
@@ -22,15 +22,17 @@ Single Table Inheritance is a trait for Laravel 5.8+ Eloquent models that allows
 Simply add the package to your `composer.json` file and run `composer update`.
 
 ```
-"nanigans/single-table-inheritance": "~1.0"
+"carsdotcom/single-table-inheritance": "^1.0.1"
```

Comment:
As soon as this merges, we'll tag a `1.0.1` release. I could be persuaded this is a new minor or major version since the runner stops testing older Laravel, but we're not actually changing the trait, so...?